test/pid_tests.cpp

```diff
@@ -168,18 +217,17 @@ TEST(ParameterTest, gainSettingCopyPIDTest)
 
   // \todo test initParam() -------------------------------------------------
 
-
   // \todo test bool init(const ros::NodeHandle &n); -----------------------------------
 
-
   // Send update command to populate errors -------------------------------------------------
   pid1.setCurrentCmd(10);
-  pid1.computeCommand(20, ros::Duration(1.0));
+  pid1.computeCommand(20, rclcpp::Duration(1.0, 0));
```

Comment:
```suggestion
  pid1.computeCommand(20, rclcpp::Duration(1, 0));
```

that constructor takes ints [afaik](http://docs.ros2.org/ardent/api/rclcpp/classrclcpp_1_1_duration.html)

Reply:
Thanks. FIxed all the tests that use `rclcpp::Duration`.